### ISSUE TITLE
Test Failure: test_code_generator.py::test_layer_multiplication

### ISSUE THREAD

**Issue by Lemniscate-SHA-256** [Lemniscate-SHA-256/Neural#69]
assert 1 == 3
 +  where 1 = <built-in method count of str object at 0x70a0494d32d0>('Dense(units=64')
 +    where <built-in method count of str object at 0x70a0494d32d0> = "import tensorflow as tf\nfrom tensorflow.keras import layers\nfrom tensorflow.keras.optimizers import adam\n\n# Model construction using Functional API\ninputs = tf.keras.Input(shape=(None, 32))\nx = inputs\nx = layers.Dense(units=64, activation='linear')(x)\nx = layers.Dropout(rate=0.5)(x)\n\nmodel = tf.keras.Model(inputs=inputs, outputs=outputs)\nmodel.compile(loss='mse', optimizer=adam())\n".count

**Comment by Lemniscate-SHA-256** [comment]
For tensorflow code generation, the problem was coming from .pop('multiply', 1), the true dict key is '*' not 'multiply'. Replaced, everything works fine.
